me.id.webverify/webverifylib/src/main/java/me/id/webverifylib/listener/IDmeGetAuthCodeListener.java

```diff
@@ -0,0 +1,9 @@
+package me.id.webverifylib.listener;
+
+/**
+ * Listener to get the auth token
+ */
+public interface IDmeGetAuthCodeListener {
```

Comment:
this is the only internal listener, maybe in the future, if we have to manage multiple internal listener we can consider it. Sounds good?